<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Productsales</class>
 <widget class="QDialog" name="Productsales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>611</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>424</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit_id"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_piece"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_sell">
      <property name="text">
       <string>Sell</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
